plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelrc/annotation/BazelrcFlagAnnotator.kt

```diff
@@ -78,18 +80,27 @@ class BazelrcFlagAnnotator : Annotator {
           .needsUpdateOnTyping()
           .create()
 
-      isOld(flag, element.textRange.substring(element.containingFile.text)) ->
+      isOld(flag, element.text) ->
         holder
-          .newAnnotation(HighlightSeverity.INFORMATION, "Flag: '${element.text}'")
+          .newAnnotation(HighlightSeverity.INFORMATION, oldAnnotationMessage(flag, element.text))
           .range(element.textRange)
           .textAttributes(BazelrcHighlightingColors.DEPRECATED_FLAG)
           .needsUpdateOnTyping()
+          // TODO(mihai): Add an intention to fix
```

Comment:
can you create a ticket for that, so we wont forget about it?

Reply:
created

Reply:
thanks!